member-service/src/main/java/com/hedvig/memberservice/services/ZignSecBankIdService.kt

```diff
@@ -31,32 +32,41 @@ class NorwegianBankIdService(
     @Value("\${redirect.authentication.failUrl}")
     private val authenticationFailUrl: String
 ) {
-    fun authenticate(memberId: Long, request: GenericBankIdAuthenticationRequest): StartNorwegianAuthenticationResult {
-        return norwegianAuthentication.auth(
-            NorwegianBankIdAuthenticationRequest(
+    fun authenticate(
+        memberId: Long,
+        request: GenericBankIdAuthenticationRequest,
+        zignSecAuthenticationMarket: ZignSecAuthenticationMarket): StartZignSecAuthenticationResult {
+        return zignSecAuthentication.auth(
```

Comment:
```suggestion
        zignSecAuthenticationMarket: ZignSecAuthenticationMarket): StartZignSecAuthenticationResult = zignSecAuthentication.auth(
```

I dont want to be that guy, BUT since we are doing it on line 55 🙈